docs/REFERENCE.md

```diff
@@ -218,6 +218,31 @@ const fetchDataOrErrorResult = Result.from(async () => {
 
 ---
 
+## Resultify
+`Resultify` is helper function very similar in work to `Promisify`. It changes return type of function to Result.
+
+**Examples:**
+
+```typescript
+import { Result, Err, Resultify } from 'type-safe-errors';
+
+class FetchFailedError extends Error {
+  name = "FetchFailedError" as const;
+}
+
+function fetchDataOrErrorResult () {
+  try {
+    const res = fetchRemoteData();
```

Comment:
did you forget `async` / `await` here?
this example seems to have much more sense when the `fetchRemoteData()` fn is async.